bin/node/runtime/pangolin/src/pallets/system/mod.rs

```diff
@@ -9,7 +9,7 @@ use sp_version::RuntimeVersion;
 use crate::*;
 
 frame_support::parameter_types! {
-	pub const BlockHashCount: BlockNumber = 2400;
+	pub const BlockHashCount: BlockNumber = 256;
```

Comment:
Here is more information about this change: https://github.com/paritytech/frontier/pull/409

Reply:
todo: create an issue to trace this change in darwinia and crab.

Reply:
Trace issue: https://github.com/darwinia-network/darwinia/issues/680